core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -561,6 +561,16 @@ sealed abstract class Parser[+A] extends Parser0[A] {
   def repSep(min: Int, max: Int, sep: Parser0[Any]): Parser[NonEmptyList[A]] =
     Parser.repSep(this, min = min, max = max, sep = sep)
 
+  /** Repeat this parser 0 or more times until `end` Parser succeeds.
+    */
+  def until0(end: Parser[Any]): Parser0[List[A]] =
```

Comment:
actually, what about `def repUntil0[B](end: Parser[Any])(implicit acc: Accumulator0[A, B]): Parser0[B]`?

similar for `def repUntil` with `Accumulator`?

Reply:
yea, I forgot about them; we should add them.